outbreak_distribution.R

```diff
@@ -0,0 +1,246 @@
+# HPRU script
+
+## Outbreak distribution size
+
+
+# Load required packages --------------------------------------------------
+
+library(epichains)
+library(ggplot2)
+library(epiparameter)
+
+# Choose a seed so results are reproducible -------------------------------
+
+set.seed(1)
+
+# Define outbreak parameter space -----------------------------------------
+
+statistic <- "size"
+offspring_dist <- "rpois"
+R <- seq(0.1, 1.0, 0.1)
+
+scenarios <- expand.grid(
+  offspring_dist = offspring_dist,
+  statistic = statistic,
+  R = R,
+  stringsAsFactors = FALSE
+)
+
+# number of simulations to run
+n_chains <- 1000
+
+# outbreak size groupings
+breaks <- c(0, 2, 5, 10, 20, 50, 100, Inf)
+
+# Simulate outbreak size distribution (Poisson) ---------------------------
+
+outbreak_list <- vector(mode = "list", length = nrow(scenarios))
+for (i in seq_len(nrow(scenarios))) {
+  offspring_dist_fun <- match.fun(scenarios[i, "offspring_dist"])
+  outbreak_list[[i]] <- epichains::simulate_chain_stats(
+    n_chains = n_chains, 
+    statistic = scenarios[i, "statistic"], 
+    offspring_dist = offspring_dist_fun,
+    lambda = scenarios[i, "R"],
+    stat_threshold = breaks[length(breaks) - 1] + 1
+  )
+}
```

Comment:
possibly add spaces within for loops to increase visibility on {epichains} code? if you agree, this may be added to all loops.
```suggestion
for (i in seq_len(nrow(scenarios))) {
  
  offspring_dist_fun <- match.fun(scenarios[i, "offspring_dist"])
  
  outbreak_list[[i]] <- epichains::simulate_chain_stats(
    n_chains = n_chains, 
    statistic = scenarios[i, "statistic"], 
    offspring_dist = offspring_dist_fun,
    lambda = scenarios[i, "R"],
    stat_threshold = breaks[length(breaks) - 1] + 1
  )
  
}
```

Reply:
added in https://github.com/epiverse-trace/epiverse_demos/pull/7/commits/d3bd4e826d31365249f6cee4ef7a0d4b1b7f89f5